borsh-derive-internal/src/attribute_helpers.rs

```diff
@@ -18,3 +23,120 @@ pub fn contains_initialize_with(attrs: &[Attribute]) -> Option<Path> {
 
```

Comment:
extract `borsh_init` to constant, add comment that it's item level attribute (struct or enum)